strax/storage/mongo.py

```diff
@@ -11,100 +11,159 @@
 import strax
 import numpy as np
 from pymongo import MongoClient
-from strax import StorageFrontend, StorageBackend, DataKey
-from strax.utils import unpack_dtype
-from bson import json_util
-import copy
+from strax import StorageFrontend, StorageBackend, Saver
 
 export, __all__ = strax.exporter()
 
 
 @export
 class MongoBackend(StorageBackend):
-
-    def __init__(self, uri, database):
+    """Mongo storage backend"""
+    def __init__(self, uri, database, col_n=None):
+        """
+        Backend for reading/writing data from Mongo
+        :param uri: Mongo url (with pw and username)
+        :param database: name of database (str)
+        :param col_n: collection name (str) to look for data
+        """
         if MongoClient is None:
```

Comment:
If PyMongo doesn't import, you'll get an `ImportError` when you try to load this file, and this line will never call. Did you mean to try-except the import statement?

Reply:
I'm also not sure why Dan coded it this way. But yeah, this is a good opportunity to clean this up. It doesn't make sense to me either